src/main/python/apache/aurora/executor/thermos_task_runner.py

```diff
@@ -121,13 +124,13 @@ def _terminate_http(self):
 
     # pass 1
     http_signaler.quitquitquit()
-    self._clock.sleep(self.ESCALATION_WAIT.as_(Time.SECONDS))
+    self._clock.sleep(self._escalation_wait.as_(Time.SECONDS))
     if self.status is not None:
       return True
 
     # pass 2
     http_signaler.abortabortabort()
-    self._clock.sleep(self.ESCALATION_WAIT.as_(Time.SECONDS))
+    self._clock.sleep(self._escalation_wait.as_(Time.SECONDS))
```

Comment:
I think that the escalation times for /qqq->/aaa and /aaa->kill should be independently configurable - every situation in which I have seen the need to configure this so far has been isolated to the /qqq->/aaa stage.